Casks/zulu-jdk25.rb

```diff
@@ -16,7 +15,6 @@
     url 'https://cdn.azul.com/zulu/bin/zulu25.30.17-ca-jdk25.0.1-macosx_aarch64.dmg',
         referer: 'https://www.azul.com/downloads/?os=macos&package=jdk#zulu'
 
-    depends_on macos: '>= :big_sur'
```

Comment:
Removed the OS version check, this diffes from https://github.com/Homebrew/homebrew-cask/blob/0533bdd4761c94b138b3691ebc3f07994ece53c2/Casks/z/zulu%4025.rb#L27. Seems no harm to land.